utils/dwarf.c

```diff
@@ -1054,9 +1153,26 @@ static void build_dwarf_info(struct debug_info *dinfo, struct symtab *symtab,
 
 		dwarf_getfuncs(&cudie, get_dwarfspecs_cb, &bd, 0);
 
+		if (dwarf_hasattr(&cudie, DW_AT_comp_dir)) {
+			comp_dir = str_attr(&cudie, DW_AT_comp_dir, false);
+			add_dir_entry(&dirs, comp_dir, dinfo->nr_locs_used);
+			dinfo->nr_locs_used = 0;
+		}
+
 		curr = next;
 	}
 
+	base_dir = get_base_dir(&dirs);
+	if (base_dir) {
+		pr_dbg3("base dir: %s\n", base_dir);
+		dinfo->base_dir = xstrdup(base_dir);
+	}
+	else {
```

Comment:
get_base_dir return first node if all cu has same count like first unit test.
if you agree with this, I'll delete this else case.
thank you.

Reply:
I'm ok with that.

Reply:
I removed else case. but I couldn't find the information that DW_AT_comp_dir is mandatory field in dwarf spec. if it's not mandatory, I have to check code in related functions. do you have related information? if we are not sure, I'll update related code to handle that case. thank you.